
### Hi, I’m Tonia 😊

I’m a Senior Bioinformatics Research Scientist at the [Bioinformatics
Core at the St. Jude Children’s Research
Hospital](https://www.stjude.org/research/departments/developmental-neurobiology/shared-resources/bioinformatic-core.html).
We provide advanced bioinformatic services for investigators to leverage
genomics, proteomics and metabolomics (omics) data.

📊 🖇️ Support project analysis

:rocket: :construction: :construction_worker: Build and automate
computational workflows and pipelines

🧬 💪 Extensive experience with WGS/WES, sc-RNA-Seq, sc-ATAC-Seq and
sc-TCR/BCR-Seq analysis in the context of tumor evolution and
microenvironment

:bug: :construction_worker: Passionate about computation, automation and
reproducibility for scientists

I am an evolutionary biologist by training with extensive expertise in
molecular evolution and phylogenetics, ecology and biogeography in the
context of species and cancer. I introduced the “Tumor Island
Biogeography” framework proposing that tumors are evolutionary islands
that their heterogeneity can be explored by using omics data in the
clinical setting ([Chroni,
2024](https://www.taylorfrancis.com/chapters/edit/10.1201/9781003307921-10/tumor-island-biogeography-antonia-chroni)
and [Chroni and Kumar, 2021](https://doi.org/10.1093/gbe/evab276)). I
co-developed [PathFinder](https://github.com/SayakaMiura/PathFinder), a
computational Bayesian method that uses the clone phylogeny, their
genetic differentiation, and anatomical locations to infer cancer’s
migration history in a patient.

For more details on my work and research, please visit my [digital
🏠](https://antoniachroni.github.io/) or my [LinkedIn
🏠](https://www.linkedin.com/in/antonia-chroni-ba1a8514b/).

### Here’s some stats on my GitHub repos

<p align="center">
<img src="https://github-readme-stats.vercel.app/api?username=AntoniaChroni&show_icons=true&theme=dracula">
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=AntoniaChroni&show=html,css,jupyter%20notebook&layout=compact">
</p>

### My recent GitHub activity

- 🗣 Commented on issue
  [stjudednbbinfcore/epigenomic-profiling-analysis#3](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis#3)
- 🤔 Opened an issue in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 😺 Deleted branch add-gitignore-file at
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 🎊 Closed an issue in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 📨 Pushed to main in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 🎉 Merged a pull request in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 🤔 Opened a pull request in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 😺 Created a branch add-gitignore-file in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 🤔 Opened an issue in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
- 😺 Created a branch main in
  [stjudednbbinfcore/epigenomic-profiling-analysis](https://github.com/stjudednbbinfcore/epigenomic-profiling-analysis)
